dowhy/causal_model.py

```diff
@@ -459,4 +460,55 @@ def summary(self, print_to_stdout=False):
             print(summary_text)
         return summary_text
 
+    def refute_graph(self, k= 1, method_name =None ):
+        """
+        Check if the dependencies in input graph matches with the dataset
+        :param k: number of covariates in set Z while testing conditional independence ( X ⫫ Y ) | Z 
+        :param method_name: name of method name to test conditional independece in data
+        
+        : returns: python lists true_implications and false_implications which contain the implications which were 
+                   were true for the dataset and not true respectively.
+        """
+        all_nodes=list(self._graph.get_all_nodes(include_unobserved=False))
+        all_possible_combinations = list(combinations(all_nodes, 2))
+        conditional_independences=[]
+        print("For the input graph following are the conditional independences: ")
+        for combination in all_possible_combinations:
+            a=combination[0]
+            b=combination[1]
+            all_nodes.remove(a) 
+            all_nodes.remove(b)
+            k_sized_lists=list(combinations(all_nodes,k)) 
+            for k_list in k_sized_lists:
+                if self._graph.check_dseparation([str(a)], [str(b)], k_list) == True :
+                    print(a , " and ",  b , " are CI given ", k_list)
```

Comment:
rather than print, use logging statements. you will examples in other functions in causal_model.py